test/RecipeRepository-test.js

```diff
@@ -1,344 +1,346 @@
-            "number": 1
-          }
-        ],
-        "name": "Maple Dijon Apple Cider Grilled Pork Chops",
-        "tags": [
-          "lunch",
-          "main course",
-          "main dish",
-          "dinner"
-        ]
-      },
-      {
-        "id": 412309,
-        "image": "https://spoonacular.com/recipeImages/412309-556x370.jpeg",
-        "ingredients": [
-          {
-            "id": 1002030,
-            "quantity": {
-              "amount": 4,
-              "unit": "teaspoons"
-            }
-          },
-          {
-            "id": 19334,
-            "quantity": {
-              "amount": 8,
-              "unit": "tablespoons"
-            }
-          },
-          {
-            "id": 1001,
-            "quantity": {
-              "amount": 2,
-              "unit": "cups"
-            }
-          },
-          {
-            "id": 4582,
-            "quantity": {
-              "amount": 4,
-              "unit": "servings"
-            }
-          },
-          {
-            "id": 2031,
-            "quantity": {
-              "amount": 4,
-              "unit": "teaspoons"
-            }
-          },
-          {
-            "id": 5100,
-            "quantity": {
-              "amount": 1,
-              "unit": "pound"
-            }
-          },
-          {
-            "id": 2009,
-            "quantity": {
-              "amount": 4,
-              "unit": "teaspoons"
-            }
-          },
-          {
-            "id": 1022020,
-            "quantity": {
-              "amount": 4,
-              "unit": "teaspoons"
-            }
-          },
-          {
-            "id": 6168,
-            "quantity": {
-              "amount": 8,
-              "unit": "cups"
-            }
-          },
-          {
-            "id": 9176,
-            "quantity": {
-              "amount": 0.5,
-              "unit": "cup"
-            }
-          },
-          {
-            "id": 2026,
-            "quantity": {
-              "amount": 4,
-              "unit": "teaspoons"
-            }
-          },
-          {
-            "id": 1042047,
-            "quantity": {
-              "amount": 1.5,
-              "unit": "tablespoons"
-            }
-          },
-          {
-            "id": 1042047,
-            "quantity": {
-              "amount": 4,
-              "unit": "teaspoons"
-            }
-          }
-        ],
-        "instructions": [
-          {
-            "instruction": "Mix the hot sauce, butter, mango habanero sauce, brown sugar, chili powder, garlic powder, onion powder, black pepper, cayenne pepper and seasoning salt in a bowl. Stir vigorously until completely combined.",
-            "number": 1
-          }
-        ],
-        "name": "Dirty Steve's Original Wing Sauce",
-        "tags": [
-          "sauce"
-        ]
-      }
-    ]
-    repository = new RecipeRepository(sampleData, ingredientsData);
+    // sampleData = [
```

Comment:
This sampling of data wasn't working for us while building out our tests- not big enough. Currently we have imported ALL the data, but as of right now, the plan is to create a new file for a larger sampling and import that in for testing.